### ISSUE TITLE
drugbank.type field should be indexed as a keyword

### ISSUE THREAD

**Issue by newgene** [biothings/mychem.info#52]
This line:
https://github.com/biothings/mychem.info/blob/a4f8d50040030adb4c0934bdd27cfb868477cbb5/src/hub/dataload/sources/pharmgkb/pharmgkb_upload.py#L74-L76

The values of drugbank.type field look like just "Drug" and "Prodrug", better to index it as `keyword`. Is it somehow recognized in the inspector as "text" field?

**Comment by sirloon** [comment]
@newgene that line show "pharmgkb.type", not "drugbank.type". As for the inspection, yes pharmgkb.type is recognized as text, not keyword

**Comment by newgene** [comment]
@sirloon you are right. It's my typo, should be "pharmgkb.type" field. Just updated.

Why it's a text field, does the inspector see a value with whitespaces?
